package.json

```diff
@@ -61,6 +61,14 @@
   "main": "lib/webpack.js",
   "web": "lib/webpack.web.js",
   "bin": "./bin/webpack.js",
+  "files": [
+    "lib/",
+    "bin/",
+    "benchmark/",
```

Comment:
Ah, ok. I just included the same ones as in the original package, other than dotfiles. Updated now :smile: